data-loader/core/src/main/java/com/scalar/db/dataloader/core/dataexport/ExportReport.java

```diff
@@ -0,0 +1,30 @@
+package com.scalar.db.dataloader.core.dataexport;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Represents the report of exported data from a table
+ *
+ * @author Jishnu J
+ */
+public class ExportReport {
+
+  /**
+   * The field is used to get the total number of rows exported from the table and written to the
+   * exported file. AtomicInteger is used because it is thread safe and this parameter is a counter
+   * which is updated inside threads
+   */
+  private final AtomicInteger exportedRowCount;
```

Comment:
`LongAdder` might be useful here:
https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/atomic/LongAdder.html

Reply:
Thank you for the suggestion.
I have replaced `AtomicInteger` with `LongAdder`.